src/sonic-build-hooks/hooks/git

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+parse_config(){
+    . /usr/local/share/buildinfo/scripts/buildinfo_base.sh
+    REAL_COMMAND=$(get_command git)
+
+    version_file=$VERSION_PATH/versions-git
+    new_version_file=$BUILD_VERSION_PATH/versions-git
+
+    MODE_CLONE=0
+    # parse input parameters
+    for i in "$@"
+    do
+        if [[ $i == "clone" ]];then
+            MODE_CLONE=1
+        fi
+    done
+}
+
+get_clone_path(){
+    # get paremater of clone path
+    while (( "$#" )); do
+        case $1 in
+            -b|--branch|--reference|--reference-if-able|-c|--config|--origin|-u|--upload-pack|-j|--jobs|--depth|--dissociate)
+                shift 2
+                ;;
+            clone|-l|--local|--no-hardlinks|-s|--shared|--dissociate|-q|--quiet|-v|--verbose|--progress|--server-option=*|--bare|--sparse|--filter=*|--template=*|--mirror|--reference|--shallow-*|--no-tags|--recurse-submodules*|--remote-submodules|--no-remote-submodules|--separate-git-dir*)
+                shift 1
+                ;;
+            *)
+                if [[ $URL == "" ]];then
+                    URL=$1
+                else
+                    clone_PATH=$1
+                fi
+                shift 1
+                ;;
+        esac
+    done
+
+    # if not specific clone path, get default clone path
+    [ -z $clone_PATH ] && clone_PATH=`echo $URL | rev | awk -F/ '{print$1}' | rev | awk -F. '{print$1}'`
```

Comment:
The command awk supports to get the last field, $NF. How about to change as below? Or any better regex by sed?
```
[ -z $clone_PATH ] && clone_PATH=`echo $URL | awk -F/ '{print$NF}' | awk -F. '{print$1}'`
```